application/controllers/QuestionAdministrationController.php

```diff
@@ -2772,6 +2772,7 @@ private function storeSubquestions($question, $subquestionsArray)
                     $subquestion = Question::model()->findByAttributes([
                         'sid' => $question->sid,
                         'parent_qid' => $question->qid,
+                        'scale_id' => $scaleId,
```

Comment:
Shouldn't you add scale id also around here: https://github.com/LimeSurvey/LimeSurvey/pull/3532/files#diff-107f8c81519e24601f4b3670bb92ccb4daa2c444134d1c14b24cce40b1e4d722R2786 ?

Reply:
Its already there just further down (line #2797).

Reply:
Sorry. My bad.